package.json

```diff
@@ -241,7 +241,7 @@
         "tslint": "^6.1.1"
     },
     "dependencies": {
-        "@microsoft/sarif-multitool": "^2.2.2",
+        "@microsoft/sarif-multitool": "^2.2.3",
```

Comment:
>    "@microsoft/sarif-multitool": "^2.2.3", [](start = 4, length = 43)

This fixes the upgrade tool not converting. Not strictly part of this PR, but it makes it work end to end.